pkg/controllers/nodeclaim/disruption/consolidation.go

```diff
@@ -34,6 +34,8 @@ type Consolidation struct {
 	clock      clock.Clock
 }
 
+// This function should only modify the nodeclaim's Status section
```

Comment:
nit: probably an unnecessary comment -- seems fine to just add the change -- it makes logical sense